### DISCUSSION TITLE
Using Discord on Figma

### DISCUSSION THREAD

**Discussion by eflight1** [tailwindlabs/tailwindcss#11749] [Help]
I purchased Tailwind but have no idea on how to get the components to show up in Figma so my designer can use them. Would appreciate help/technical direction to make this happen. Thank you.

**Comment by Kennetht43** [comment]
Hello, 
I have suggest to answer share this topic, you can read this.
To integrate Tailwind components into Figma for your designer to use, you can follow these steps:

Open the Tailwind Plugin:
After installation, open your Figma file.
In the left sidebar, you'll find the "Plugins" section.
Click on the "Tailwind CSS" plugin to open it.

Generate Components:
Once the Tailwind plugin is open, you can start generating components.
Select an element or group in your Figma design that you want to convert into a Tailwind component.
In the plugin, click on the "Generate Component" button. It will analyze the selected element and convert it into corresponding Tailwind CSS classes.

Copy the Tailwind Classes:
The plugin will display a list of Tailwind CSS classes that represent the selected component.
Click on the "Copy" button next to the classes you want to use for that component.

Apply the Tailwind Classes:
Go back to your Figma design, select the element or group you want to style with the Tailwind classes.
In the right sidebar, under the "Design" tab, you'll find the "Text" or "Effects" section, depending on what you are styling.
Click on the input box for "Effects" or "Text" to add the Tailwind classes you copied. Paste them here.

Repeat for Other Components:
Continue generating and applying Tailwind components to other elements in your design as needed.
Please note that while the Tailwind plugin can help you create and apply CSS classes to elements in Figma, it might not fully replicate all aspects of Tailwind CSS as used in web development. Some customizations may require manual adjustments.

Additionally, make sure your designer has a basic understanding [TellPopeyes Survey Code](https://www.tellpopeyes.me/) of Tailwind CSS and its utility classes so they can make the most out of the generated components.

If you encounter any issues specific to the Tailwind Figma plugin or need more detailed instructions, it's best to consult the plugin's documentation or reach out to the plugin developer for support.

Thanks for read this information, i hope my information is good and helpful.

Best wish

**Comment by adamwathan** [comment]
Hey @eflight1! As mentioned on the Tailwind UI website, Tailwind UI is just a code product and does not include Figma assets:

<img width="791" alt="image" src="https://github.com/tailwindlabs/tailwindcss/assets/4323180/74e27fbb-5220-4283-bb76-e8325a89f98d">

If that's a deal breaker for you shoot an email to support@tailwindui.com for a refund 👍
